client/blocks/authentication/social/social-tos.jsx

```diff
@@ -29,29 +27,14 @@ function getToSComponent( content ) {
 }
 
 function SocialAuthToS( props ) {
-	let content = props.translate(
-		'If you continue with Google or Apple, you agree to our {{tosLink}}Terms of Service{{/tosLink}}, and have read our {{privacyLink}}Privacy Policy{{/privacyLink}}.',
+	const content = props.translate(
+		'If you continue with Google, Apple or GitHub, you agree to our {{tosLink}}Terms of Service{{/tosLink}}, and have read our {{privacyLink}}Privacy Policy{{/privacyLink}}.',
```

Comment:
cc @TimBroddin since you implemented "Continue with GitHub" feature. I believe we should include "GitHub" in the ToS but let me know if I'm wrong.